frontend/src/slices/orders/selectors.ts

```diff
@@ -1,5 +1,20 @@
-import { type RootState } from '~/libs/types/types.js';
+import { type DataStatus } from '~/libs/enums/data-status.enum.js';
+import {
+  type OrderResponseDto,
+  type RootState,
+  type ValueOf,
+} from '~/libs/types/types.js';
+
+import { type RouteData } from './libs/types/types.js';
 
 const selectPrice = (state: RootState): number => state.orders.price;
 
 export { selectPrice };
+
+const selectOrder = (state: RootState): (OrderResponseDto | undefined)[] =>
+  state.orders.orders;
```

Comment:
```suggestion
const selectOrder = (state: RootState): (OrderResponseDto | undefined)[] => {
return  state.orders.orders;
}
```
[B27](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md)